### ISSUE TITLE
libicu requirements for distress containers

### ISSUE THREAD

**Issue by cohenjo** [microsoft/delta-kusto#89]
Hi, 
I'm trying to run delta-kusto in a minimal distress container.

and got the following error when running delta-kusto:
```
Couldn't find a valid ICU package installed on the system. Please install libicu using your package manager and try again. Alternatively you can set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support. 
Please see https://aka.ms/dotnet-missing-libicu for more information.
```

I tried running again with `export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1` as suggested but hit the following error:
```
Unhandled exception. System.Globalization.CultureNotFoundException: Only the invariant culture is supported in globalization-invariant mode. 
See https://aka.ms/GlobalizationInvariantMode for more information. (Parameter 'name')
en-us is an invalid culture identifier
```

Can you please confirm if libicu installation is indeed required or if a different solution exists for this.

The docker build:
```
FROM gcr.io/distroless/cc-debian11:nonroot-debug
WORKDIR /
COPY --from=builder /lib/x86_64-linux-gnu/libz.so.1 /lib/x86_64-linux-gnu/libz.so.1
COPY --from=builder /lib/x86_64-linux-gnu/libgcc_s.so.1 /lib/x86_64-linux-gnu/libgcc_s.so.1
COPY --from=builder /tmp/delta-kusto /bin/
```

(I added lib and libgcc as they showed up as missing when I ran ldd on delta-kusto - will not start without them...)

**Comment by cohenjo** [comment]
I found a solution, writing here for anyone else trying to run this in a container.

There's a base image containing required dependencies for dotnet applications (even self-contained ones)

https://github.com/dotnet/dotnet-docker/blob/6392ade69cf6d8ba1790cdd8fa7e6c42d2d6613c/README.runtime-deps.md

And a Dockerfile for those who want to look into the details:
https://github.com/dotnet/dotnet-docker/blob/6392ade69cf6d8ba1790cdd8fa7e6c42d2d6613c/src/runtime-deps/6.0/cbl-mariner1.0-distroless/amd64/Dockerfile

**Comment by vplauzon** [comment]
Fixed in [0.8.2](https://github.com/microsoft/delta-kusto/releases/tag/0.8.2.102)
